core/market/src/config.rs

```diff
@@ -33,37 +35,61 @@ pub struct SubscriptionConfig {
     pub default_ttl: chrono::Duration,
 }
 
-#[derive(Clone)]
+#[derive(StructOpt, Clone)]
 pub struct EventsConfig {
+    #[structopt(env = "MAX_MARKET_EVENTS_DEFAULT", default_value = "20")]
     pub max_events_default: i32,
+    #[structopt(env = "MAX_MARKET_EVENTS_MAX", default_value = "100")]
     pub max_events_max: i32,
 }
 
+#[derive(StructOpt, Clone)]
+pub struct DbConfig {
+    /// Interval in which Market cleaner will be invoked
+    #[structopt(env = "MARKET_DB_CLEANUP_INTERVAL", parse(try_from_str = humantime::parse_duration), default_value = "24h")]
+    pub cleanup_interval: Duration,
+    /// Number of days to persist Agreements and related Agreement Events
+    #[structopt(env = "MARKET_AGREEMENT_STORE_DAYS", default_value = "90")]
```

Comment:
Why drop the `YAGNA_` prefix?

Reply:
Besides the default, there was also the `min` value (30). Will it be ignored now?

Reply:
> Why drop the `YAGNA_` prefix?

To be consistent with the rest of env vars here.

> Besides the default, there was also the `min` value (30). Will it be ignored now?

I was struggling with this decision but finally decided to drop this feature.